apple/OmnivoreKit/Sources/Views/Resources/zh-Hans.lproj/Localizable.strings

```diff
@@ -20,6 +20,7 @@
 "labelsViewAssignNameColor" = "指定名称和颜色。";
 "createLabelMessage" = "创建新标签";
 "labelsPurposeDescription" = "使用标签创建精选的链接集合。";
+"labelNamePlaceholder" = "标签名称";
```

Comment:
Please, feel free to correct me here, I don't really know what I am doing 🤣

Reply:
haha, asked and its right!